Tools/Frame_params/iflight-chimera7.param

```diff
@@ -0,0 +1,77 @@
+#NOTE: iFlight Chimera7 params for ArduPilot Copter 4.2
+
+ACRO_RP_EXPO,0.6
+ACRO_RP_RATE,600
+ACRO_Y_EXPO,0.6
+ACRO_Y_RATE,600
+ATC_ACCEL_P_MAX,281062.5
+ATC_ACCEL_R_MAX,301679.8
+ATC_ACCEL_Y_MAX,72000
+ATC_ANG_PIT_P,18
+ATC_ANG_RLL_P,18
+ATC_ANG_YAW_P,11
+ATC_RAT_PIT_D,0.001879093
+ATC_RAT_PIT_FLTD,60
+ATC_RAT_PIT_FLTT,30
+ATC_RAT_PIT_I,0.09663188
+ATC_RAT_PIT_P,0.09663188
+ATC_RAT_RLL_D,0.001453624
+ATC_RAT_RLL_FLTD,75
+ATC_RAT_RLL_FLTT,30
+ATC_RAT_RLL_I,0.07706726
+ATC_RAT_RLL_P,0.07706726
+ATC_RAT_YAW_D,0.02
+ATC_RAT_YAW_FF,0.0043
+ATC_RAT_YAW_FLTD,60
+ATC_RAT_YAW_FLTE,1.887445
+ATC_RAT_YAW_FLTT,30
+ATC_RAT_YAW_I,0.01927038
+ATC_RAT_YAW_P,0.1927038
+BATT_AMP_OFFSET,-0.04308
+BATT_AMP_PERVLT,52.706
+BATT_CURR_PIN,11
+BATT_LOW_VOLT,21.6
+BATT_MONITOR,4
+COMPASS_AUTO_ROT,0
+COMPASS_CUS_PIT,-15
+COMPASS_CUS_ROLL,0
+COMPASS_CUS_YAW,180
+COMPASS_ENABLE,1
+COMPASS_EXTERNAL,1
+COMPASS_ORIENT,100
+COMPASS_USE,1
+EK3_PRIMARY,1
+EK3_IMU_MASK,2
+FRAME_CLASS,1
+FRAME_TYPE,12
+INS_GYRO_FILTER,150
+INS_HNTCH_BW,20
+INS_HNTCH_ENABLE,1
+INS_HNTCH_FREQ,40
+INS_HNTCH_HMNCS,1
+INS_HNTCH_MODE,3
+INS_HNTCH_OPTS,6
+INS_HNTCH_REF,1
+INS_HNTC2_BW,75
+INS_HNTC2_ENABLE,1
+INS_HNTC2_FREQ,150
+INS_HNTC2_HMNCS,1
+INS_HNTC2_MODE,1
+INS_HNTC2_REF,1
+MOT_BAT_VOLT_MAX,25.2
+MOT_BAT_VOLT_MIN,19.8
+MOT_PWM_TYPE,6
+MOT_THST_EXPO,0.62
+MOT_THST_HOVER,0.12
+PILOT_Y_EXPO,0.4
+PILOT_Y_RATE,600
+PSC_ACCZ_I,0.24
+PSC_ACCZ_P,0.12
+RC2_REVERSED,1

```

Comment:
Are you sure you want this RC2_REVERSED? I'd say this depends upon the transmitter used.  Of course if it comes with a specific transmitter then this might be ok.

Reply:
Yeah, I'm conflicted. This is what is required with CRSF. I've never quite understood why this is - is it because of the way AP works, or CRSF works or something else?

Reply:
Ok removed